ios/engine/KMEI/KeymanEngine/Classes/Model/KMPKeyboard.swift

```diff
@@ -83,7 +86,7 @@ public class KMPKeyboard
                                            languageID: language.languageId,
```

Comment:
I think the loop in which this code belongs is where we should be breaking out, so we are only adding a single `InstallableKeyboard` to the list to be installed but the language data is there for other potential uses in the future.

Reply:
There is now a .first instead of a for each language in when installing keyboards, so it only installs the keyboard for its first language. But every keyboard in the package gets installed.